packages/token/src/common/erc2981/erc2981.cairo

```diff
@@ -109,6 +115,175 @@ pub mod ERC2981Component {
         }
     }
 
+    #[embeddable_as(ERC2981InfoImpl)]
+    impl ERC2981Info<
+        TContractState,
+        +HasComponent<TContractState>,
+        +ImmutableConfig,
+        +SRC5Component::HasComponent<TContractState>,
+        +Drop<TContractState>,
+    > of interface::IERC2981Info<ComponentState<TContractState>> {
+        /// Returns the royalty information that all ids in this contract will default to.
+        ///
+        /// The returned tuple contains:
+        ///
+        /// - `t.0`: The receiver of the royalty payment.
+        /// - `t.1`: The numerator of the royalty fraction.
+        /// - `t.2`: The denominator of the royalty fraction.
+        fn default_royalty(self: @ComponentState<TContractState>) -> (ContractAddress, u128, u128) {
+            InternalImpl::default_royalty(self)
+        }
+
+        /// Returns the royalty information specific to a token.
+        /// If no specific royalty information is set for the token, the default is returned.
+        ///
+        /// The returned tuple contains:
+        ///
+        /// - `t.0`: The receiver of the royalty payment.
+        /// - `t.1`: The numerator of the royalty fraction.
+        /// - `t.2`: The denominator of the royalty fraction.
+        fn token_royalty(
+            self: @ComponentState<TContractState>, token_id: u256
+        ) -> (ContractAddress, u128, u128) {
+            InternalImpl::token_royalty(self, token_id)
```

Comment:
Internal functions clashing with external functions should underscored:
Applies for the rest of the file as well.

```suggestion
            self._token_royalty(token_id)
```

Reply:
Underscored all internal functions except `initializer`